client/src/main/webjar/add-program/add-program.component.js

```diff
@@ -1,34 +0,0 @@
-(function (angular) {
```

Comment:
I renamed the "add-program" components to "create-program" components, use the word "create" instead of "add", and use the material design icon for "create" instead of "add".

The "add" icon is a plus sign, and on a map this can be confused as "zoom in". 

The create icon is a pencil, so no such confusion should take place.